python/mit/binding.py.in

```diff
@@ -167,12 +195,12 @@ libmit.mit_destroy.argtypes = [c_void_p]
 libmit.mit_register_args.argtypes = [c_void_p, c_int, c_void_p]
 
 libmit.mit_align.restype = c_uword
```

Comment:
This should have a thing to convert it to a Python int.

Reply:
Only if we do that for everything returning a `mit_WORD`.